### PR TITLE
[12.0][MIG] users_ldap_mail

### PR DESCRIPTION
**Pull request by Tardo**
cc @tecnativa TT18055

### CODE REVIEW

File: users_ldap_mail/readme/USAGE.rst [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+#. Open Odoo in your browser
+#. Go to Settings
+#. Go to General Settings
+#. In The General Settings form go to LDAP Parameters
+#. In a LDAP Parameters item there are two new fields: mail and name, the name
+   These items will correspond with a new user that is created, when a user
+   logs in via LDAP in Odoo.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
```
Comment (chienandalu) [inline users_ldap_mail/readme/USAGE.rst]: I think this can go out, as there's a badge already for it generated by the oca-gen-readme

**Review by chienandalu — COMMENTED** [review]
Would it be very hard to achieve full coverage?

File: users_ldap_mail/__init__.py [hunk 2]
```diff
@@ -1,5 +1,4 @@
-# © Daniel Reis (https://launchpad.com/~dreis-pt)
+# Copyright (C) Daniel Reis (https://launchpad.com/~dreis-pt)
```
Comment (pedrobaeza) [inline users_ldap_mail/__init__.py]: No need of the `(C)`

**Review by pedrobaeza — COMMENTED** [review]
Can you squash a bit the commit history?

**Comment by Tardo** [comment]
@chienandalu The module only overwrite 'email' and 'name' values of the res.users record... Make tests for this module isn't a trivial task...

All other changes done!

**Review by chienandalu — APPROVED** [review]

**Comment by chienandalu** [comment]
Ok, great :)

**Pull request merged**